stdlib/typing.pyi

```diff
@@ -744,9 +749,21 @@ else:
         obj: _get_type_hints_obj_allowed_types, globalns: dict[str, Any] | None = None, localns: dict[str, Any] | None = None
     ) -> dict[str, Any]: ...
 
+def get_args(tp: Any) -> tuple[Any, ...]: ...
+
 if sys.version_info >= (3, 8):
```

Comment:
Looks like you accidentally moved the `get_args` function outside of the `>= (3, 8)` block here (it only exists in `typing` on 3.8+)

```suggestion
if sys.version_info >= (3, 8):
    def get_args(tp: Any) -> tuple[Any, ...]: ...
```

Reply:
Yep, no worries. Still in progress :)